DEVELOPMENT.md

```diff
@@ -0,0 +1,72 @@
+# Development
+
+This doc explains how to setup a development environment so you can get started
+[contributing](https://www.knative.dev/contributing/) to `Knative Test Infra`.
+Also take a look at:
+
+- [The pull request workflow](https://www.knative.dev/contributing/contributing/#pull-requests)
+- [Iterating](#iterating)
+
+## Prerequisites
+
+Before submitting a PR, see also [CONTRIBUTING.md](./CONTRIBUTING.md).
+
+### Sign up for GitHub
+
+Start by creating [a GitHub account](https://github.com/join), then setup
+[GitHub access via SSH](https://help.github.com/articles/connecting-to-github-with-ssh/).
+
+### Checkout your fork
+
+The Go tools require that you clone the repository to the
```

Comment:
This is actually not a requirement anymore after we switched to Go modules.

Reply:
Does this mean the `path_alias` in prow jobs isn't required either?

Reply:
I _think_ it's not required either, but I'm not entirely sure. We may need to do some quick experiment to see if that's the case.

Reply:
I've made a Prow job without path_alias - https://prow.knative.dev/view/gcs/knative-prow/logs/ci-knative-serving-continuous-test-noalias/1280208283721797632, let's see how it goes.

Reply:
Looks like this means someone will need to download knative.dev/test-infra for building the code, which is odd to me:
```
go: downloading knative.dev/test-infra v0.0.0-20200630141629-15f40fe97047
```

Reply:
nvm, didn't notice the example was serving repo. Updated the instruction